src/loki/status_action.cc

```diff
@@ -1,8 +1,40 @@
+#include <boost/property_tree/json_parser.hpp>
+#include <iostream>
+#include <sstream>
+
+#include "baldr/tilehierarchy.h"
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+  // get _some_ tile
+  static baldr::graph_tile_ptr tile = nullptr;
+  if (!tile) {
+    for (const auto& tile_id : reader->GetTileSet()) {
+      tile = reader->GetGraphTile(tile_id);
+      if (tile->id().level() < baldr::TileHierarchy::GetTransitLevel().level &&
+          tile->header()->nodecount() > 0) {
```

Comment:
I had some (bit older) tiles flying around which had tons of empty 2.level tiles reaching all the way to the antimeridian. Not sure if that was caused by an older valhalla bug which is fixed now or by slightly wrong Geofabrik PBFs. So I added this check, can't hurt I guess

Reply:
they probably werent empty they probably had bins in them. its totally possible to have a long ferry that doesnt have any graph nodes along it for a long long stretch over the ocean. at the antimeridian is weird but its not buggy to find a tile with no nodes or edges in it but just bins

Reply:
yeah I suspect faulty osm data here

Reply:
This was #3202